examples/03_renderable/main.rs

```diff
@@ -82,7 +83,7 @@ impl<'a> System<'a> for ExampleSystem {
 struct Example;
 
 impl State for Example {
-    fn on_start(&mut self, engine: &mut Engine) {
+    fn on_start<'a, 'b>(&mut self, engine: &mut Engine, _: &mut Scene) -> Option<DispatcherBuilder<'a, 'b>> {
```

Comment:
That's the exact opposite of what I asked for earlier. Why not have one very flexible dispatcher? After all, one could even wrap a dispatcher in a system and add it.